app/controllers/managers/users_controller.rb

```diff
@@ -3,13 +3,10 @@ class Managers::UsersController < ManagersController
   before_action :employee, only: :update
 
   def index
-    @users = User.by_name(params[:name])
-                 .by_email(params[:email])
-                 .by_group(params[:group])
-                 .by_role(params[:role])
-                 .by_status(params[:status])
-                 .sort_by_created_at(:desc)
-                 .page(params[:page]).per Settings.page.size
+    @q = User.ransack(params[:q])
```

Comment:
a ko muốn để là params[:q] đuowjc ko e

Reply:
params key mặc định cho search params hiện tại là :q, nó thay thế cho params[:search] lúc trước để rút ngắn các chuỗi truy vấn. Có thể custom một cái tên khác như thế này ạ:
`ruby
# in the controller:
@search = Log.ransack(params[:log_search], search_key: :log_search)
# or
@search = Log.search(params[:log_search], search_key: :log_search)
` 
`ruby
in the view:
<%= f.search_form_for @search, as: :log_search %>
<%= sort_link(@search) %>
`